config/skyrat/skyrat_config.txt

```diff
@@ -72,3 +72,6 @@ PC_MOB_TEXT As a player controlled mob you are expected to play the role to the
 ## Disables any ERP preferences for codebases that don't want it.
 #DISABLE_ERP_PREFERENCES
 #ERP_EMOTES_TO_DISABLE cum
+
+## Disables the automatic kicking system
+#DISABLE_AFK_AUTOKICK
```

Comment:
This should probably be "ENABLE_AFK_AUTOKICK" or something, negative variable names are usually not the best way to go around doing things.

Reply:
Hmm, we have a habit of leaving config changes out :)